core/src/main/scala/org/apache/spark/streaming/eventhubs/EventHubsClientWrapper.scala

```diff
@@ -131,22 +132,27 @@ private[eventhubs] class EventHubsClientWrapper extends Serializable with EventH
     // Create Eventhubs client
     eventhubsClient = EventHubClient.createFromConnectionStringSync(connectionString)
 
+    // Enabling flag to receive runtime info from eventhub client.
+    val receiverOptions = new ReceiverOptions
+    receiverOptions.setReceiverRuntimeMetricEnabled(true)
```

Comment:
What if the customer only have 32 bytes payload...how many bytes we received once we turned on for every batch?